src/constants/MarkdownFiles/posts/2025-06-08-dmp-25-AmanNaik-week01.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "DMP ’25 Week 1 Update by Aman Naik"
+excerpt: "This week's focus was exploring the write-activity codebase, finding appropriate grammar correction models & understanding Abiword documentations."
+category: "DEVELOPER NEWS"
+date: "2025-06-08"
+slug: "2025-06-08-dmp-25-AmanNaik-week01"
+author: "@/constants/MarkdownFiles/authors/amannaik247.md"
+description: "Member and DMP'25 Contributor at SugarLabs"
+tags: "dmp25,writeactivity,write,sugarlabs,week01,amannaik247"
+image: "assets/Images/c4gt_DMP.png"
+---
+
+<!-- markdownlint-disable -->
+
+# Week 1 Progress Report by Aman Naik
+
+**Project:** [Add an AI-assistant to the Write Activity](https://github.com/sugarlabs/write-activity/issues/52)  
+**Mentors:** [Walter Bender](https://github.com/walterbender), [Ibiam Chihurumnaya](https://github.com/chimosky)  
```

Comment:
There seems to be something happening here:

![Screenshot from 2025-06-15 08-03-07](https://github.com/user-attachments/assets/d1a435a2-14cb-45e5-b22d-6bba7e6e2076)

Perhaps try adding a space to see if that forces the new line?